.travis.yml

```diff
@@ -7,7 +7,7 @@ cache:
     - $HOME/.opam
 env:
   global:
-    - PACKAGE=ounit EXTRA_DEPS=oasis OUNIT_CI=true
+    - PACKAGE=ounit EXTRA_DEPS=dune OUNIT_CI=true
```

Comment:
Drop the extra deps, I should be a normal dependency in ounit.opam (this one was a trick because setup.ml was using DynRun, so we needed oasis to build the dev version, but releases were using full setup.ml).